_posts/2021-11-01-r-6.0.0.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: Apache Arrow R 6.0.0 Release
+date: "2021-11-01"
+author: 
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Grouped aggregation
+
+Aggregations can now be made across groups using dplyr’s `group_by() %>% summarise()` syntax. Arrow 5.0.0 allowed `summarise()` to aggregate across a whole dataset, but 6.0.0 now allows you to aggregate across groups with `group_by()`. These are usable both with in-memory Arrow tables as well as across partitioned datasets. As usual, Arrow will read and process data in chunks and in parallel when possible to produce results much faster than one could by loading it all into memory then processing and even better, allows for operations that wouldn’t fit into memory on a single machine.
```

Comment:
I don't think 5.0.0 supported summarize at all, did it?

Reply:
I think it did - https://github.com/apache/arrow/pull/10191 - the release was like July, right?

Reply:
IIRC it supported summarize only for non-grouped tables. So one could do `table %>% summarise()` and get one row of some aggregates for the whole table but not `table %>% group_by() %>% summarise()`

Reply:
Oops, re-reading the paragraph in the post and not just the comments (sorry about not doing that before!) @thisisnic is absolutely right here. Here's `summarize` from the 5.0.0 release tag](https://github.com/apache/arrow/blob/release-5.0.0/r/R/dplyr-summarize.R#L21-L36). 

I think we can close this comment.